docs/shared/overview/supplierInit.mdx

```diff
@@ -118,6 +122,14 @@ If at some point, you feel like products magically disappear from Mealz recipes,
 from the app basket while they should be, this is probably related to this part.
 :::
 
+### Helper Function to map between products
+
+More than likely, your Product object will be quite different than our `SupplierProduct`.
```

Comment:
More than likely, your Product object will be quite different than our, then we have create a bridge model call `SupplierProduct`

Reply:
c'est fait